ansible/roles/jitsi/meta/main.yml

```diff
@@ -0,0 +1,4 @@
+---
+dependencies:
+  - nftables
```

Comment:
I don't know that this actually needs to be a dependency, it will just update when we run the nftables role and that's fine.

NGINX should be as you'll need to use the handlers (see other comment)

Reply:
I just thought that we'd run the role only when we've had all the preconfiguration necessary, i can remove it if you'd like